gitlab/util.go

```diff
@@ -70,6 +71,19 @@ func stringToVisibilityLevel(s string) *gitlab.VisibilityValue {
 	}
 	return &value
 }
```

Comment:
missing whiteline

Reply:
I think my VSCode tries to do something different than `make fmt`. I pushed code again and CI is passing. I'm missing something ?

Reply:
well you should run go fmt + add a blank line here :)

Reply:
I think we should be good now :)